src/main/java/edu/harvard/iq/dataverse/DatasetFieldConstant.java

```diff
@@ -157,6 +158,8 @@ public class DatasetFieldConstant implements java.io.Serializable  {
     public final static String confidentialityDeclaration="confidentialityDeclaration";
     public final static String specialPermissions="specialPermissions";
     public final static String restrictions="restrictions";
+    @Deprecated
+    //Doesn't appear to be used and is not datasetContact
     public final static String contact="contact";
```

Comment:
Huh. Good catch. I suspect you're right. I suppose we could mark the getContact method as deprecated as well.

Reply:
done